### PR TITLE
adding note about the fullscreen directive bug

### PR DESCRIPTION
**Pull request by chrisdavidmills**
As per https://bugzilla.mozilla.org/show_bug.cgi?id=1608358

### CODE REVIEW

File: http/headers/feature-policy.json [hunk 1]
```diff
@@ -638,7 +638,8 @@
               },
               "firefox": [
                 {
-                  "version_added": "74"
+                  "version_added": "74",
+                  "notes": "Originally, applying <code>fullscreen</code> to an <code>&lt;iframe&gt;</code> (i.e. via <code>allow</code>) didn't work unless the <code>allowfullscreen</code> attribute was also present. This was fixed in Firefox 80."
```
Comment (ddbeck) [inline http/headers/feature-policy.json:642]: Style suggestions

```suggestion
                  "notes": "Before Firefox 80, applying <code>fullscreen</code> to an <code>&lt;iframe&gt;</code> (i.e. via the <code>allow</code> attribute) does not work unless the <code>allowfullscreen</code> attribute is also present."
```

**Review by ddbeck — DISMISSED** [review]
Thank you, Chris! Some style suggestions for you, but otherwise it looks good.

**Comment by ddbeck** [comment]
I know Chris is too busy to come back to this soon, so I've applied my suggestion and I'm going take ownership of this.

@sideshowbarker, perhaps you could review this?

**Comment by sideshowbarker** [comment]
> @sideshowbarker, perhaps you could review this?

Yes — I’ve set myself as the assignee for now (to ensure it stays in my TODO list until I do the actual review (later today or tomorrow my time)

**Review by sideshowbarker — APPROVED** [review]
LGTM, after reading through the discussion in https://bugzilla.mozilla.org/show_bug.cgi?id=1608358

**Pull request merged**